### PR TITLE
Creating awesome books website using plain js classes 

### PR DESCRIPTION
**Pull request by MMGGYY66**
- Use JavaScript classes instead of objects and arrays.
 - Create class methods to add and remove books.
- Add CSS styles to the application to make it match this wireframe:

### CODE REVIEW

File: assets/js/app.js [hunk 1]
```diff
@@ -1,92 +1,146 @@
...
+
+// display the  Contact section when click the button "Contact"
+const contactBtn = document.querySelector('.contact');
+const contactInfo = document.querySelector('.contact-info');
 
-  removeBook(position);
+contactBtn.addEventListener('click', () => {
+  bookList.style.display = 'none';
+  formContainer.style.display = 'none';
+  contactInfo.style.display = 'block';
```
Comment (anagudelogu) [inline assets/js/app.js:145]: - [ ] I notice that you have a lot of code that you're not using. At the moment these lines don't serve any kind of purpose, please remove them in order to keep your codebase clean and readable. 🤓 🛠️

File: assets/js/app.js [hunk 2]
```diff
@@ -1,92 +1,146 @@
...
+let books;
+
+// SavedBooks Class: Handles Storage
+class SavedBooks {
+  static getBooks() {
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
```
Comment (anagudelogu) [inline assets/js/app.js:17]: - [ ] **[OPTIONAL]:** You could achieve the same in one line using any of the [fallback operators](https://salferrarello.com/javascript-or-versus-nullish-coalescing-operator/). As shown below:

```suggestion
    books = JSON.parse(localStorage.getItem('books')) || []
```

This means that if there's a key in LocalStorage called 'books', set the value of the books variable to its content, else fallback to an empty array. 🤓 🛠️

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @MMGGYY66, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @MMGGYY66 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good PR description :heavy_check_mark: 
- Professional README :heavy_check_mark: 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**